praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
```

Comment:
You should add some tests :P. If it's any trouble just do whatever tests are done for the popular subreddits, and alter that a tiny bit to fit, and run the tests to make the betamax cassette. But regardless, all tests seem to be currently broken unfortunately since betamax updated.